go/tao/auth/parser.go

```diff
@@ -345,13 +345,13 @@ func (p *parser) expectTerm() (Term, error) {
 		if !ok {
 			return nil, fmt.Errorf("a keyword must be a string")
 		}
```

Comment:
Are not using s here, so there is no need for that variable or the check. You can delete these 4 lines.